ui.apps/src/main/content/jcr_root/apps/acs-commons/config/org.apache.sling.serviceusermapping.impl.ServiceUserMapperImpl.amended-acs-commons.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:sling="http://sling.apache.org/jcr/sling/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+          jcr:primaryType="sling:OsgiConfig"
+          user.default=""
+          user.mapping="[com.adobe.acs.acs-aem-commons-bundle:ensure-oak-index=[acs-commons-ensure-oak-index-service],com.adobe.acs.acs-aem-commons-bundle:email-service=[acs-commons-email-service],com.adobe.acs.acs-aem-commons-bundle:httpcache-jcr-storage-service=[acs-commons-httpcache-jcr-storage-service],com.adobe.acs.acs-aem-commons-bundle:review-task-asset-mover=[acs-commons-review-task-asset-mover-service],com.adobe.acs.acs-aem-commons-bundle:error-page-handler=[acs-commons-error-page-handler-service],com.adobe.acs.acs-aem-commons-bundle:form-helper=[acs-commons-form-helper-service],com.adobe.acs.acs-aem-commons-bundle:dispatcher-flush=[acs-commons-dispatcher-flush-service],com.adobe.acs.acs-aem-commons-bundle:package-replication-status-event-listener=[acs-commons-package-replication-status-event-service],com.adobe.acs.acs-aem-commons-bundle:component-error-handler=[acs-commons-component-error-handler-service],com.adobe.acs.acs-aem-commons-bundle:system-notifications=[acs-commons-system-notifications-service],com.adobe.acs.acs-aem-commons-bundle-twitter:twitter-updater=[acs-commons-twitter-updater-service],com.adobe.acs.acs-aem-commons-bundle:workflow-remover=[acs-commons-workflow-remover-service],com.adobe.acs.acs-aem-commons-bundle:bulk-workflow=[acs-commons-bulk-workflow-servic],com.adobe.acs.acs-aem-commons-bundle:bulk-workflow-runner=[workflow-process-service],com.adobe.acs.acs-aem-commons-bundle:ensure-service-user=[acs-commons-ensure-service-user-service],com.adobe.acs.acs-aem-commons-bundle:shared-component-props=[acs-commons-shared-component-props-service],com.adobe.acs.acs-aem-commons-bundle:manage-controlled-processes=[acs-commons-manage-controlled-processes-service],com.adobe.acs.acs-aem-commons-bundle:automatic-package-replicator=[acs-commons-automatic-package-replicator-service],com.adobe.acs.acs-aem-commons-bundle:on-deploy-scripts=[acs-commons-on-deploy-scripts-service],com.adobe.acs.acs-aem-commons-bundle:remote-assets=[acs-commons-remote-assets-service],com.adobe.acs.acs-aem-commons-bundle:workflowpackagemanager-service=[acs-commons-workflowpackagemanager-service],com.adobe.acs.acs-aem-commons-bundle:redirect-manager=[acs-commons-manage-redirects-service],com.adobe.acs.acs-aem-commons-bundle:file-fetch=[acs-commons-file-fetch-service]]"/>
```

Comment:
Is it possible for this to be stored on multiple lines to ease up on merge conflicts?

Reply:
yeh - good idea.